library/core/src/iter/traits/iterator.rs

```diff
@@ -2851,7 +2851,7 @@ pub trait Iterator {
         Product::product(self)
     }
 
-    /// Lexicographically compares the elements of this [`Iterator`] with those
+    /// [Lexicographically](core::cmp::Ord#lexicographical-comparison) compares the elements of this [`Iterator`] with those
```

Comment:
Oops, this is in the `core` crate, which can't use `core` since crates don't know their own name (https://github.com/rust-lang/rust/issues/73445). Fortunately `Ord` is already in scope so you can use that directly.
```suggestion
    /// [Lexicographically](Ord#lexicographical-comparison) compares the elements of this [`Iterator`] with those
```
Same for the rest of the links in core, you'll get a full list when the build fails.

Reply:
Got it. Thanks!

Reply:
I will try `cargo-deadlinks` locally.

Reply:
> I will try cargo-deadlinks locally.

No need for that, CI runs the linkchecker for you. You can test with `x.py doc library/std` that the intra-doc links resolve, though.